src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -157,24 +163,56 @@ public String getBasePropertyImport(String moduleName) {
     return moduleName + "." + EXPORT_PROPERTY_NAME;
   }
 
+  public boolean isCommonJsImport(Node requireCall) {
+    return isCommonJsImport(requireCall, compiler.getOptions().getModuleResolutionMode());
+  }
+
   /**
-   * Recognize if a node is a module import. We recognize the form:
+   * Recognize if a node is a module import. We recognize two forms:
    *
    * <ul>
    *   <li>require("something");
+   *   <li>__webpack_require__(4); // only when the module resolution is WEBPACK
    * </ul>
    */
-  public static boolean isCommonJsImport(Node requireCall) {
+  public static boolean isCommonJsImport(
+      Node requireCall, ModuleLoader.ResolutionMode resolutionMode) {
     if (requireCall.isCall() && requireCall.hasTwoChildren()) {
-      if (requireCall.getFirstChild().matchesQualifiedName(REQUIRE)
+      if (resolutionMode == ModuleLoader.ResolutionMode.WEBPACK
+          && requireCall.getFirstChild().matchesQualifiedName(WEBPACK_REQUIRE)
+          && requireCall.getSecondChild().isNumber()) {
+        return true;
+      } else if (requireCall.getFirstChild().matchesQualifiedName(REQUIRE)
           && requireCall.getSecondChild().isString()) {
         return true;
       }
+    } else if (requireCall.isCall()
+        && requireCall.getChildCount() == 3
+        && resolutionMode == ModuleLoader.ResolutionMode.WEBPACK
+        && requireCall.getFirstChild().isQualifiedName()
+        && requireCall.getFirstChild().matchesQualifiedName(WEBPACK_REQUIRE + ".bind")
+        && requireCall.getSecondChild().isNull()
+        && requireCall.getChildAtIndex(2).isNumber()) {
+      return true;
     }
     return false;
   }
 
-  public static String getCommonJsImportPath(Node requireCall) {
+  public String getCommonJsImportPath(Node requireCall) {
+    return getCommonJsImportPath(requireCall, compiler.getOptions().getModuleResolutionMode());
+  }
+
+  public static String getCommonJsImportPath(
+      Node requireCall, ModuleLoader.ResolutionMode resolutionMode) {
+    if (resolutionMode == ModuleLoader.ResolutionMode.WEBPACK) {
+      if (requireCall.getSecondChild().isNumber()) {
+        return String.valueOf(Double.valueOf(requireCall.getSecondChild().getDouble()).intValue());
```

Comment:
Here and below, we can simplify
   Double.valueOf(n.getDouble()).intValue()
->
  (int) n.getDouble()